apps/chatbots/views.py

```diff
@@ -187,3 +202,24 @@ def start_authed_web_session(request, team_slug: str, experiment_id: int, versio
 
 def chatbot_invitations(request, team_slug: str, experiment_id: int):
     return experiment_invitations(request, team_slug, experiment_id, "chatbots")
+
+
+def start_chatbot_session_public(request, team_slug: str, experiment_id: uuid.UUID):
```

Comment:
All 'team' views should have either `login_and_team_required` or the `team_required` decorator:

```suggestion
@team_required
def start_chatbot_session_public(request, team_slug: str, experiment_id: uuid.UUID):
```